cc/toolchains/args/archiver_flags/BUILD

```diff
@@ -1,10 +1,22 @@
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 load("//cc/toolchains:args.bzl", "cc_args")
 load("//cc/toolchains:args_list.bzl", "cc_args_list")
 load("//cc/toolchains:feature.bzl", "cc_feature")
 load("//cc/toolchains:nested_args.bzl", "cc_nested_args")
 
 package(default_visibility = ["//visibility:private"])
 
+bool_flag(
+    name = "use_libtool_on_macos",
+    build_setting_default = True,
+)
+
+config_setting(
+    name = "use_libtool_on_macos_setting",
```

Comment:
Should these targets be public to allow toolchain implementors to select on it?

Reply:
Hrm what would be the use case? I guess if you're avoiding this feature maybe you should Add your own flag? Not sure

Reply:
Basically this; https://github.com/bazelbuild/rules_cc/pull/403 i am not sure what's the best approach here.